fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/streams/Streamer.java

```diff
@@ -19,7 +19,7 @@
 
 /**
  * Simple implementation taking a JDBC ResultSet and rendering it using
- * the Java 8 Streams API.
+ * the Java 11 Streams API.
```

Comment:
maybe better to just take it out in this case
```suggestion
 * the Java Streams API.
```

Reply:
Updated.